### ISSUE TITLE
[regression] incorrect setting of replacement for DOT_FONTNAME and DOT_FONTSIZE

### ISSUE THREAD

**Issue by albert-github** [doxygen/doxygen#9482]
When having, with the current master (1.9.5 (1e5aa121826c53e50b2e290efaa3f8bcd19bfb7c)) the simple `Doxyfile`:
```
DOT_FONTNAME           = Helvetica
DOT_FONTSIZE           = 20
HAVE_DOT = YES
CALL_GRAPH=YES
CALLER_GRAPH=YES
```
we get with the command `doxygen -x`: 
```
warning: Tag 'DOT_FONTNAME' at line 1 of file 'Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOT_FONTSIZE' at line 2 of file 'Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
# Difference with default Doxyfile 1.9.5 (1e5aa121826c53e50b2e290efaa3f8bcd19bfb7c)
HAVE_DOT               = YES
DOT_COMMON_ATTR        = "fontname=Helvetica fontsize=10 fontname=Helvetica fontsize=20"
DOT_EDGE_ATTR          = "labelfontname=Helvetica labelfontsize=10 labelfontname=Helvetica labelfontsize=20"
CALL_GRAPH             = YES
CALLER_GRAPH           = YES
```
The both warnings are correct, but the duplication is not as the wrong, in this case size, is chosen.
This is nicely illustrated with the simple source file:
```
/// \file

/// \brief first fie
/// \details first fie
int fie_1()
{
  fie_2();
}

/// \brief second fie
/// \details second fie
int fie_2()
{
}
```

With the doxygen 1.9.4 version we see:
![image](https://user-images.githubusercontent.com/5223533/182019291-9c302475-475f-4e49-a12d-1509a7cccb6d.png)

and with the mentioned master version we get:
![image](https://user-images.githubusercontent.com/5223533/182019306-817a9542-34bb-4fe3-97a6-d635364a851d.png)

Note with the 1.9.4. version the size is 20 and with the current master 10.

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/9228553/example.tar.gz)

**Comment by albert-github** [comment]
@makelinux regression due to #9463

**Comment by doxygen** [comment]
@albert-github Please check if this problem has been properly fixed with https://github.com/doxygen/doxygen/commit/a44f6a7e5caf1fbf5fe271c99046a51b315d5f56. If so, this issue can be closed, as it was introduced after 1.9.4 was released.

**Comment by albert-github** [comment]
I did a few tests and the problem looks like to be fixed.
